tox.ini

```diff
@@ -5,15 +5,13 @@ small_test_ee = quay.io/ansible/python-base
 mypy_tmp_dir = mypy_tmp_dir
 
 [tox]
-envlist = linters, type, py38, report, clean
+envlist = linters, type, py, report, clean
```

Comment:
I'm not sure if this is needed. It was clearly selected to signal what interpreter is currently expected. If we want more, we need to start with the CI first.

Reply:
Previous value was incorrect, especially for a project that officially supports multiple versions of python. Listing specific version on default envlist is a mistake. As others said the version agnostic "py" should be used here.